style.css

```diff
@@ -0,0 +1,86 @@
+header {color: #3498db; font-size: 20px; background-color: green;}
+a {
+  content: "first
+    second";
+  }
+ a {
+  content: "first
+    second";
+}
+
+a {
+  font-family: "Times
+    New
+    Roman";
+}
+
+a {
+  content: "first\Asecond";
+}
+a {
+  content: "first\\nsecond";
+}
+[title="nothing\
+  is wrong"] {}
+
+
+
+    a {
+  colr: blue;
+}
+a {
+  my-property: 1;
+}
+
+
+
+a {
+  color: green;
+}
+a {
+  fill: black;
+}
+
+
+@keyframes important2 {
+  from { margin: 10px }
+  to { margin: 20px !important }
+}               
+
+
+@keyframes important1 {
+  from {
+    margin-top: 50px;
+  }
+  to {
+    margin-top: 100px !important;
+  }
+}
+
+
+
+a { color: pink !important; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/WebHeroSchool/testtest/pullRequest?prid=5323395)